tesseract/tesseract_planning/tesseract_process_managers/include/tesseract_process_managers/process_input.h

```diff
@@ -73,6 +90,12 @@ struct ProcessInput
   /** @brief Global Manipulator Information */
   const ManipulatorInfo& manip_info;
 
+  /** @brief Planner Plan Profile mapping */
+  const PlannerProfileMapping& plan_profile_mapping;
+
+  /** @brief Planner Composite Profile mapping */
+  const PlannerProfileMapping& composite_profile_mapping;
+
```

Comment:
I'm not sure I would put this in the process input.  While I understand wanting to put it in the Motion Planning request so we don't have to have multiple copies of the planner with different mappings, I think I would prefer that each generator has a set of mappings and we keep the ProcessInput simple and independent of the processes it is passed into.

Reply:
This also means that if these mappings were loaded from a file, that would happen here at the process manager rather than at the caller.

Reply:
Based on our discussion, I renamed things and updated the documentation in the ProcessInput class.